src/objects.rs

```diff
@@ -1440,6 +1440,62 @@ impl AddPrefix for TicketUseRestriction {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GridCalendars {
+    pub id: String,
+    pub name: String,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub monday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub tuesday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub wednesday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub thursday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub friday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub saturday: bool,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub sunday: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GridExceptionDates {
+    pub calendar_id: String,
+    #[serde(
+        deserialize_with = "de_from_date_string",
+        serialize_with = "ser_from_naive_date"
+    )]
+    pub date: Date,
+    #[serde(deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
+    pub r#type: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GridPeriods {
+    pub calendar_id: String,
+    #[serde(
+        deserialize_with = "de_from_date_string",
+        serialize_with = "ser_from_naive_date"
+    )]
+    pub begin_date: Date,
+    #[serde(
+        deserialize_with = "de_from_date_string",
+        serialize_with = "ser_from_naive_date"
+    )]
+    pub end_date: Date,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct GridRelCalendarLine {
+    pub calendar_id: String,
+    #[serde(skip)]
```

Comment:
Not sure I understand why this field is skipped?

Reply:
files in production don't have this field